### DISCUSSION TITLE
Show current Azure Subscription only when activating az command?

### DISCUSSION THREAD

**Discussion by SavaMihai** [JanDeDobbeleer/oh-my-posh#537] [Troubleshoot] [answered]
Hello,
Is it possible to do this? It's more like how oh my zsh is acting. Instead of always displaying the current set Azure Subscription on the prompt to show it when I actually type "az"

**Comment by tradiff** [comment, marked as answer]
If you're referring to PowerShell, It's not really possible for a prompt to do that in PowerShell because the prompt function is short-lived and ends after the prompt is displayed.
My [✨Magic Tooltips✨](https://github.com/TravisTX/MagicTooltips/) module attempts to fill this gap.
